src/api/route-services/experiment.js

```diff
@@ -246,7 +246,7 @@ class ExperimentService {
     return prettyData;
   }
 
-  async savePipelineHandle(experimentId, handle) {
+  async saveHandle(experimentId, handle, service) {
```

Comment:
In the part of the code dealing with state machines, this `service` concept is called `processName` (well, sort of, because it takes the values `qc` and `gem2s`).  For consistency, I would use that also here, but I see that it would create a new inconsistency in the accepted values. 

Having said that, since this is internal to the file, I think that I am splitting hairs :-)
https://github.com/biomage-ltd/api/blob/44cd2eeaec115d9b820cb6eba010adf3881d0fe5/src/api/general-services/pipeline-manage/index.js#L58